test/python/WMCore_t/BossAir_t/BasePlugin_t.py

```diff
@@ -66,10 +66,15 @@ def testScramArchtoRequiredArch(self):
             bp.scramArchtoRequiredArch("slc7_BLAH_gcc700")
 
         self.assertEqual(bp.scramArchtoRequiredArch(['slc5_amd64_gcc481', 'slc6_amd64_gcc630']), 'X86_64')
-        self.assertEqual(bp.scramArchtoRequiredArch(['slc7_amd64_gcc10', 'slc7_aarch64_gcc700']), 'X86_64')
-        self.assertEqual(bp.scramArchtoRequiredArch(
-                ['slc7_amd64_gcc10', 'slc7_aarch64_gcc700', 'slc7_ppc64le_gcc9']), 'X86_64')
-        self.assertEqual(bp.scramArchtoRequiredArch(['slc7_aarch64_gcc700', 'slc7_ppc64le_gcc9']), 'ppc64le')
+        test = (bp.scramArchtoRequiredArch(['slc7_amd64_gcc10', 'slc7_aarch64_gcc700'])).split(',')
```

Comment:
I think one pair of parenthesis here is not really needed (the first one).

Reply:
I think I got an error with ` .spllit()` without the parenthesis